mantidimaging/core/io/saver.py

```diff
@@ -23,6 +23,11 @@
     from ..data.imagestack import ImageStack
     from ..utility.data_containers import Indices
 
+try:
+    from mantidimaging.versions import package_version  # type: ignore
+except ImportError:
+    package_version = '0.0.0.dev1'
+
```

Comment:
Could you use `CheckVersion.get_version()` from `mantidimaging.core.utility.version_check`